vmdb/app/controllers/ops_controller/settings/common.rb

```diff
@@ -326,6 +326,11 @@ def settings_update_save
       @edit[:new].set_worker_setting!(:MiqWebServiceWorker, :count, w[:count].to_i)
       @edit[:new].set_worker_setting!(:MiqWebServiceWorker, :memory_threshold, human_size_to_rails_method(w[:memory_threshold]))
 
+      w = qwb[:automate_worker]
+      @edit[:new].set_worker_setting!(:MiqAutomateWorker, :count, w[:count].to_i)
+      @edit[:new].set_worker_setting!(:MiqAutomateWorker, :memory_threshold,
+                                      human_size_to_rails_method(w[:memory_threshold]))
+
       @update = MiqServer.find(@sb[:selected_server_id]).get_config
```

Comment:
@mkanoor @dclarizio @martinpovolny Is there any way to refactor the duplication?